lib/shopify_api/resources/inventory_item.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+module ShopifyAPI
+  class InventoryItem < Base
+  end
+end
```

Comment:
To follow the same pattern other .rb files are following, I'd consider adding an extra line over here.

Reply:
Most of these actually have new lines at the end. Github for some reason truncates the last line of the file if its blank (which is really weird?). I did forget the new line on another file though and github displays those with a little red warning icon 

[![alt](https://screenshot.click/07-03-18-11-07-77evs-vi1y9.jpg)](https://screenshot.click/07-03-18-11-07-77evs-vi1y9.jpg)